assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -1431,9 +1431,11 @@ export default Component.extend({
     });
   },
 
-  _handle429Errors(error) {
+  _handleErrors(error) {
```

Comment:
Is this function going to handle more errors based on statuses? If you think that’s the case then maybe we could use a `switch` statement instead? Something like that:
```javascript
switch (error?.jqXHR?.status) {
  case 429:
  case 404:
    popupAjaxError(error);
  default:
    throw error;
}
```

Reply:
I don't know, but the switch doesn’t hurt :+1: